diff --git a/build/lib/driver.js b/build/lib/driver.js
index 2c41640be82d900b75177b49f7a24f4d3c0d5e58..2e7efb2641be7dfad5eb031c1a6734c6f34168c2 100644
--- a/build/lib/driver.js
+++ b/build/lib/driver.js
@@ -36,7 +36,7 @@ const viewport_1 = require("./commands/viewport");
 const execute_method_map_1 = require("./execute-method-map");
 // The range of ports we can use on the system for communicating to the
 // UiAutomator2 HTTP server on the device
-const DEVICE_PORT_RANGE = [8200, 8299];
+const DEVICE_PORT_RANGE = [8200, 8999];
 // The guard is needed to avoid dynamic system port allocation conflicts for
 // parallel driver sessions
 const DEVICE_PORT_ALLOCATION_GUARD = support_1.util.getLockFileGuard(node_path_1.default.resolve(node_os_1.default.tmpdir(), 'uia2_device_port_guard'), { timeout: 25, tryRecovery: true });
