diff --git a/lib/runner/tasks.js b/lib/runner/tasks.js
index 0a690dec3a89def08a11bedda600af28c0c1b040..e02b2c5d8907713f506caecef057b80b836c4080 100644
--- a/lib/runner/tasks.js
+++ b/lib/runner/tasks.js
@@ -47,7 +47,7 @@ var import_path = __toESM(require("path"));
 var import_util = require("util");
 var import_utils = require("playwright-core/lib/utils");
 var import_utilsBundle = require("playwright-core/lib/utilsBundle");
-var import_dispatcher = require("./dispatcher");
+var import_dispatcher = require("../../../../run/playwright-custom/dispatcher");
 var import_failureTracker = require("./failureTracker");
 var import_loadUtils = require("./loadUtils");
 var import_projectUtils = require("./projectUtils");
